specs/electra/beacon-chain.md

```diff
@@ -545,6 +548,22 @@ def is_partially_withdrawable_validator(validator: Validator, balance: Gwei) ->
     )
 ```
 
+#### New `is_eligible_for_partial_withdrawals`
+
+```python
+def is_eligible_for_partial_withdrawals(validator: Validator, balance: Gwei) -> bool:
+    """
+    Check if ``validator`` can process a pending partial withdrawal.
+    """
+    has_sufficient_effective_balance = validator.effective_balance >= MIN_ACTIVATION_BALANCE
+    has_excess_balance = balance > MIN_ACTIVATION_BALANCE
+    return (
+        validator.exit_epoch == FAR_FUTURE_EPOCH
+        and has_sufficient_effective_balance
+        and has_excess_balance
+    )
```

Comment:
In a follow up PR, I would like to rename:

* `is_partially_withdrawable_validator` -> `has_withdrawable_excess_balance`
* `is_eligible_for_partial_withdrawals` -> `is_partially_withdrawable_validator`

The "partially withdrawable" function is very misleading now.